packages/ui-kit-components/src/components/PageNotFound/PageNotFound.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Paper, Typography } from '@mui/material';
```

Comment:
Some of the elements on the page begin to overlap on smaller screen sizes.  
![image](https://user-images.githubusercontent.com/106989342/203353397-a9f877b7-8d40-4f1f-8141-a1d03aec4279.png)

Reply:
The Sadface icon might be too big for mobile screens, and the layout will need to change slightly. I'll check with Nate on what should be done here.